script.js

```diff
@@ -1,5 +1,202 @@
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[1]}${playerArray[0]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
+}
+
+// 3 conditions: 
+// a > b
+// b < a
+// a == b
+var gameOutcome = function () {
+  if (Number(player1diceResults.join("")) > Number(player2diceResults.join(""))) {
+    myOutputValue = `Congratulations ${player1} for winning!!! <br><br>
+    ${player1} BEATS ${player2} with a score of ${Number(player1diceResults.join(""))} against ${Number(player2diceResults.join(""))} <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  } 
+  else if (Number(player2diceResults.join("")) > Number(player1diceResults.join(""))) {
+    myOutputValue = `Congratulations ${player2} for winning!!! <br><br>
+     ${player2} BEATS ${player1} with a score of ${Number(player2diceResults.join(""))} against ${Number(player1diceResults.join(""))}<br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+  else if (Number(player2diceResults.join("")) == Number(player1diceResults.join(""))){
+    myOutputValue = `Issa Draw lubba dubba dub dub <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+};
+
+
+
+
+currentGameMode = "Welcome to Beat That"
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  console.log("test");
-  return myOutputValue;
+  if (currentGameMode == "Welcome to Beat That"){
+    myOutputValue = `Welcome to BEAT THAT!<br><br>
+     Player 1 please enter your name and click ENTER, followed by player 2 name and click ENTER to start the game. <br><br>
+      Game Rules: <br>
+      1. This is a 2 person, player vs player game. <br>
+      2. The game will roll 2 dice and show the dice rolls result. For example, dice 1 rolled 3 and dice 2 rolled 4. <br>
+      3. The player that rolled the dice will pick the order of the dice.(If they wanted the number 43, they would specify that the 2nd dice goes first.) <br>
+      4. After both players have rolled and chosen their dice order, the player with the higher combined number wins.  `;
+
+    currentGameMode = "Enter Player 1 name";
+    console.log(`currentGameMode: ${currentGameMode}`);
+
+    return myOutputValue;
+  }
+  // Take in player 1 and 2 names
+  if (currentGameMode == "Enter Player 1 name"){
+    player1 = input
+    currentGameMode = "Enter Player 2 name"
+    console.log(`Player 1: ${player1}`);
+    myOutputValue = `Welcome ${player1}. <br> Player 2, please enter your name click enter. `;
+    console.log(`currentGameMode: ${currentGameMode}`);
+    return myOutputValue
+  }
+  if (currentGameMode == "Enter Player 2 name"){
+    player2 = input
+    currentGameMode = "Player 1 turn"
+    console.log(`Player 2: ${player1}`);
+    myOutputValue = `Welcome ${player2}. <br> Player 1, press enter to start your roll `;
+    console.log(`currentGameMode: ${currentGameMode}`);
+    return myOutputValue
+  }
```

Comment:
this is ok but a bit repetitive, consider how you can refactor this so that there is less code repetition

one way i can think of is having a `playerNames` array instead of the `player1` and `player2` variables you currently have then keep pushing new player names into the array until the desired amount of players is reached, upon which you switch the game state to 'Player 1 turn'